message MSG_CLIENT_DM_GROUP_ADMIN_DESTROY
{
    optional string token = 1;
    optional string userName = 2;
}

message MSG_DM_CLIENT_GROUP_ADMIN_DESTROY_ACK
{
    optional int32 statusCode = 1;
}